Composer/packages/lib/code-editor/src/utils/common.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+export function processSize(size) {
+  return !/^\d+$/.test(size) ? size : `${size}px`;
+}
+
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+export function noop() {}
+
+export function assignDefined(target, ...sources) {
```

Comment:
can you provide a comment in the code to what this does and why it is needed?

Reply:
these functions was imported at previous commit attempt implement our own `react-monaco-editor`.  I can make a clean up